Extension/bin/edge_strings.cs.json

```diff
@@ -0,0 +1 @@
 vyskytovat před příponou l nebo L v literálu s pevnou desetinnou čárkou.","Možnost fixed_point se dá použít jenom při kompilaci C.","Celočíselný operand může způsobit přetečení typu s pevnou desetinnou čárkou.","Konstanta s pevnou desetinnou čárkou je mimo rozsah.","Hodnota s pevnou desetinnou čárkou se nedá přesně prezentovat.","Konstanta je pro typ long long příliš velká, zadaný typ je long long bez znaménka (nestandardní).","Kvalifikátor rozložení nemůže kvalifikovat ukazatel na sdílený typ void.","duplicitní THREADS v typu vícedimenzionálního pole","Silná direktiva using se může vyskytovat jenom v rozsahu oboru názvů.","%nf deklaruje funkci jinou než funkci šablony – pokud chcete odkazovat na instanci šablony, přidejte <>.","Operace může způsobit přetečení typu s pevnou desetinnou čárkou.","Výraz musí být integrálového nebo výčtového typu nebo typu s pevnou desetinnou čárkou.","Výraz musí být integrálového typu nebo typu s pevnou desetinnou čárkou.","Funkce deklarovaná pomocí noreturn se vrací.","Název asm se ignoruje, protože je v konfliktu s předchozí deklarací.","U definice typu člena třídy se nedá změnit deklarace.","převzetí adresy temporary","Atributy se ignorují u deklarace třídy, která není zároveň definicí.","Hodnota s pevnou desetinnou čárkou se implicitně převedla na typ s plovoucí desetinnou čárkou.","Typy s pevnou desetinnou čárkou nemají žádnou klasifikaci.","Parametr šablony nemůže být typem s pevnou desetinnou čárkou.","Šestnáctkové konstanty s plovoucí desetinnou čárkou nejsou povolené.","Možnost named_address_spaces se dá použít jenom při kompilaci C.","Hodnota s plovoucí desetinnou čárkou se nevejde do požadovaného typu s pevnou desetinnou čárkou.","Hodnota se nedá přesně převést na hodnotu s pevnou desetinnou čárkou.","Výsledkem konverze na pevnou desetinnou čárku je změna znaménka.","Celočíselná hodnota se nevejde do požadovaného typu s pevnou desetinnou čárkou.","Výsledek operace s pevnou desetinnou čárkou je mimo rozsah.","několik pojmenovaných adresních prostorů","Proměnná s automatickým trváním úložiště nemůže být uložená v pojmenovaném adresním prostoru. ","Typ se nemůže kvalifikovat pomocí pojmenovaného adresního prostoru.","Typ funkce se nemůže kvalifikovat pomocí pojmenovaného adresního prostoru.","Typ pole se nemůže kvalifikovat pomocí pojmenovaného adresního prostoru.","Hodnota s pevnou desetinnou čárkou se nevejde do požadovaného typu s plovoucí desetinnou čárkou.","Hodnota s pevnou desetinnou čárkou se nevejde do požadovaného celočíselného typu.","Hodnota se nevejde do požadovaného typu s pevnou desetinnou čárkou.","Možnost named_registers se dá použít jenom při kompilaci C.","Třída úložiště pojmenovaného registru tady není povolená.","Došlo ke změně deklarace %nd s nekompatibilní třídou úložiště pojmenovaného registru.","Třída úložiště pojmenovaného registru se nedá specifikovat pro proměnnou alias.","Specifikátor úložiště pojmenovaného registru se už používá.","Možnost embedded_c se nedá kombinovat s možnos
```

Comment:
Can these JSON files be formatted with spaces/newlines?

Reply:
I'd rather not.  These are generated, and not intended to be edited by hand.  Also, they are rather large, and formatting would increase their size.

Reply:
Bob suggested the same.  I made them more readable.